frontend/src/features/recommendation/RecommendationApi.tsx

```diff
@@ -9,7 +9,8 @@ export const getRecommendedVideos = async (
   const conversionTime = new Map();
   const params = new URLSearchParams(searchString);
   conversionTime.set('Any', 1);
-  conversionTime.set('Today', dayInMillisecondes);
+  // "Today" is 36 hours to get most of the videos from the previous day
+  conversionTime.set('Today', dayInMillisecondes * 1.5);
```

Comment:
Note that the publication datetime of a video is currently truncated to keep the date only (i.e the hours/minutes/seconds are not preserved). I don't see any reason for not keeping the exact publication date (maybe legacy reasons I am not aware of?) 

So it may also be relevant in another PR to update the metadata mechanism to save the exact datetime and to make this filter behave more consistently.

Reply:
I see. I was not aware and I don't think there are any reason for this. That's part of the reason reason why we very rarely saw videos in the "Today" filtered list.

Reply:
Will be solve by #818